src/04-arrays.js

```diff
@@ -79,7 +91,10 @@ function getLastElementOfArray(array) {}
  *  addElementToEndOfArray([ 10 ], 9);
  *  //> [ 10, 9 ]
  */
-function addElementToEndOfArray(array, element) {}
+function addElementToEndOfArray(array, element) {
+  array.push(element)
```

Comment:
How come we can't just return ` array.push(element) `?  It's good to know why we that won't work here.